src/main/java/seedu/address/logic/commands/client/AddClientCommand.java

```diff
@@ -39,7 +39,9 @@ public class AddClientCommand extends ClientCommand {
             + PREFIX_PROJECT_ID + "1";
 
     public static final String MESSAGE_SUCCESS = "New client added: %1$s";
-    private static final String MESSAGE_CLIENT_ALREADY_PRESENT = "This project already has a client";
+    private static final String MESSAGE_PROJECT_LIST_UPDATED = "This client already heads other project(s) and now "
```

Comment:
This is an enhancement which increases the specificity of the success message. Not allowed in v1.4.